### ISSUE TITLE
learning commons pages need to be intergrated

### ISSUE THREAD

**Issue by gjwehnes** [thirstyice/abe-site#120]
content has been added in learning-commons-integration branch (as a .ZIP file)

-pages and content need to be modified to fit our template
-decision needs to be made on where in the website these pages will be placed. they will either be their own supercategory (learning commons) or as a category under student services. as there will eventually be substantial content here, I would recommend the former, but suggestions are welcome.

**Comment by thirstyice** [comment]
Area. Student services is too big already

**Comment by thirstyice** [comment]
Modifications aren't going to cut it. This has to be re-done from scratch.

**Comment by kid47** [comment]
@thirstyice, please leave this for others to do. We have a couple people from my comp sci class who are looking to get in by foing this as a pull request and we need to teach them. We can then help them along from the comments

**Comment by thirstyice** [comment]
I think this is too big to leave to someone who is unfamiliar with editing the website.

**Comment by kid47** [comment]
I have. We have multiple people who want to do it, and there's very few issues that can be worked on like this at the moment. We have to teach them somehow and at least now there's the opportunity to monitor it

**Comment by thirstyice** [comment]
I will watch them like a hawk

**Comment by kid47** [comment]
I know you will. I was counting on that. Just be gentle in the comments

**Comment by owmygreenmatter** [comment]
hi all,
in the Library Web Page html mr. roberts attempts to link both the Library Pathways submenus to a folder containing multiple pdfs. not entirely sure of his intentions there, but is the best way to implement those pdfs by using iframes?

**Comment by kid47** [comment]
I would say so, just as long as it makes sense content wise (as in they're not obviously meant to a part of a different section)

**Comment by thirstyice** [comment]
When is work going to start on this?

**Comment by kid47** [comment]
Currently in progress

**Comment by thirstyice** [comment]
Where?

**Comment by kid47** [comment]
In a fork

**Comment by thirstyice** [comment]
None of the forks have any changes

**Comment by kid47** [comment]
Ah. I'm not entirely sure then. I know there's progress being made however as I heard about it in comp sci

**Comment by thirstyice** [comment]
Can it be pushed so that it can be reviewed?

**Comment by kid47** [comment]
I'll ask

**Comment by owmygreenmatter** [comment]
progress on the first draft is being made at a good rate. iirc there are only two unfinished pages remaining (both Library Pathways links)--do you want me to just push it without those two?

**Comment by kid47** [comment]
I think that would be a good idea, and then we can check it out before anything is set in stone

**Comment by thirstyice** [comment]
For future reference, big changes like this should probably be done over several small commits, so that if there is a problem we can revert only that commit, and also to be more descriptive about what exactly was done.

**Comment by owmygreenmatter** [comment]
took me more fumbling with the UI than i care to admit but the changes are now on my fork, under the learning-commons-integration branch. and you're absolutely right, thirstyice, in future i'll go about big changes like that. cheers for the advice

**Comment by kid47** [comment]
Doesn't appear to have been pushed as of now

**Comment by owmygreenmatter** [comment]
it's there for me--were you checking under the learning-commons-integration branch?

**Comment by kid47** [comment]
Never mind. I'm doing this on my phone, so functionality is reduced

**Comment by owmygreenmatter** [comment]
ok, phew. it's still possible i'm using this website correctly

**Comment by kid47** [comment]
Yea, you're good to go

**Comment by thirstyice** [comment]
Ok, reviewing the code here are some issues:
Section headers should be `<h3>`, not `<h2><u>` [here, for example](https://github.com/owmygreenmatter/abe-site/blob/learning-commons-integration/b829/aberhart/learning-commons/learning-commons.html#L39)
[Guideline reference](https://github.com/thirstyice/abe-site/wiki/Guidelines#headers-and-text)

This comment will be edited when I have more time

**Comment by owmygreenmatter** [comment]
whoops, you're absolutely right! i've fixed the section header and will be awaiting further instructions

**Comment by thirstyice** [comment]
Further instructions: Remember to commit and push after making changes

**Comment by owmygreenmatter** [comment]
ok, all your main issues should be fixed. before i commit and push, do you want me to change the table in learning-commons.html or is nothing really wrong with it?

**Comment by owmygreenmatter** [comment]
ok, pushed and commited it and now all of your main issues should be fixed. not sure what to you want me to do with the table in learning-commons.html, although i do agree that it stands out. any suggestions?

**Comment by thirstyice** [comment]
I don't see your push anywhere

**Comment by kid47** [comment]
Check the network graph

**Comment by thirstyice** [comment]
I did; there's the original push, but no updates

**Comment by kid47** [comment]
The update showed up for me... I'm not really sure what happened

**Comment by owmygreenmatter** [comment]
sorry, i made a mistake there. it's definitely been pushed now

**Comment by thirstyice** [comment]
The work experience section could probably be merged with the career center page in student services. (Consult @gjwehnes )
Staff contact info could be at the top of the learning commons main page.
The learning commons main page could be grouped into tabs (see student services)
The link to New arrivals/dvds shouldn't exist if we have nothing to link to.

**Comment by thirstyice** [comment]
Is work still happening on this?

**Comment by owmygreenmatter** [comment]
work is still happening, yes, although it did come to a halt during spring break. 
sorry for the lack of updates--i've incorporated most of the changes you suggested up there in a commit on my master branch, in case you're interested in seeing what's been done. i believe @gjwehnes will be showing those same changes to mr. roberts sometime in the near future, to see what gets his approval and what doesn't.
also, whether or not to merge pages with the career center is a tricky decision, as i'm sure you're aware, so we've been trying to get everything else perfect before we worry about that.

**Comment by gjwehnes** [comment]
all changes have been merged into learning-commons-integration branch (https://github.com/thirstyice/abe-site/commit/ab71dfd01974dd4560c85538693c41ce3cad35b1)

As mentioned, this should still be reviewed by 1) our client (mr. Roberts) and 2) all members of this team.

The hope is for this to go live by the end of this week (24 April)

**Comment by Bigbobo558** [comment]
I have given the pages created by Mr. owmygreenmatter and I think the design along with the functionality of the web pages are quite well done. In my personal opinion, these should be added to the official website.

**Comment by gjwehnes** [comment]
Merged into master branch :-)

Well done owmygreenmatter for sticking with it, and to all for your support and guidance, This was as much a learning experience in workflow (i.e. how to develop and integrate a fairly major addition to our site <i>in parallel</i> to the ongoing maintenance.

**Comment by thirstyice** [comment]
Not finished yet. See #143

**Comment by owmygreenmatter** [comment]
@gjwehnes thank you very much, i'm really glad to see it finally go up.

@thirstyice i'm sorry but i don't follow; i know you made a separate branch but i'm not sure what should happen there. could you please elaborate?

**Comment by thirstyice** [comment]
The new branch included only those commits which were relevant to the implementation of learning-commons.
